module/src/components/image/image.basic.scss

```diff
@@ -0,0 +1,24 @@
+.arm-image {
+  position: relative;
+
+  .arm-status {
+    padding: $spacing-small;
+    width: 100%;
+    height: 100%;
+    border: $line-light;
+    @include absolute-fill;
+    @include flex-centre-content;
```

Comment:
Are these mixins documented? They sound pretty useful so would be good to have a list of them and what they do somewhere in storybook (if not already there)

Reply:
We talked about it at that meeting at the start, and the outcome was to write a ticket to go through them all and document them, adding them to Storybook as we go